### PR TITLE
Feature/google sign in

### LINKED ISSUE #1: Implement Google Sign In
Send user profile information to backend (mocked URL by now)

### PR DESCRIPTION
**Pull request by mduenast**

### CODE REVIEW

**Comment by mduenast** [comment]
We need to add support for circleci @rarques

File: public/index.html [hunk 1]
```diff
@@ -7,14 +7,18 @@
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="Web site created using create-react-app"
+      content="play your online role-playing games with your friends without leaving home."
     />
+    <meta name="google-signin-scope" content="profile email">
+    <meta name="google-signin-client_id" content="129422048752-f6c7ajp6j4pk7s9u6gkrcohjpaasq430.apps.googleusercontent.com">
```
Comment (rarques) [inline public/index.html]: The GOOGLE_CLIENT_ID is defined in the [GoogleBtn component](https://github.com/garan-no-dou/role-ui/pull/3/files?file-filters%5B%5D=.css&file-filters%5B%5D=.js&file-filters%5B%5D=.json#diff-8e1040cfc10cfa7eac6a5d521baf1e26R6), do we need to define it as well in the index.html?

File: src/components/GoogleSignin/GoogleAuthenticationService.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import axios from 'axios';
+import GoogleAuthenticationDTO from "./GoogleAuthenticationDTO";
+
+class GoogleAuthenticationService {
+
+    constructor() {
+    }
+
+    createSession(response) {
```
Comment (rarques) [inline src/components/GoogleSignin/GoogleAuthenticationService.js]: It's strange to call this method `createSession` when the name of the class has nothing to do with the Session concept. Why not calling it authenticate?
Also, calling the parameter `response` is confusing. What about calling it `googleLoginResponse`?

**Review by rarques — CHANGES_REQUESTED** [review]

**Comment by rarques** [comment]
I created [this issue ](https://github.com/garan-no-dou/role-ui/issues/5) to integrate CircleCI in the pipeline. In this MR we can ignore the failing checks.

File: public/index.html (hunk 1 above)
Comment (mduenast) [reply public/index.html]: resolved

File: src/components/GoogleSignin/GoogleAuthenticationService.js (hunk 2 above)
Comment (mduenast) [reply src/components/GoogleSignin/GoogleAuthenticationService.js]: resolved

**Comment by mduenast** [comment]
> 
> 
> I created [this issue ](https://github.com/garan-no-dou/role-ui/issues/5) to integrate CircleCI in the pipeline. In this MR we can ignore the failing checks.

okay

**Review by rarques — APPROVED** [review]

**Pull request merged**